pallets/rent-machine/src/lib.rs

```diff
@@ -277,6 +285,26 @@ pub mod pallet {
             let renter = ensure_signed(origin)?;
             Self::relet_machine_by_block(renter, rent_id, relet_duration)
         }
+
+        #[pallet::call_index(4)]
+        #[pallet::weight(Weight::from_parts(10000, 0))]
+        pub fn bond_evm_address(
+            origin: OriginFor<T>,
+            machine_id: MachineId,
+            evm_address: H160,
+        ) -> DispatchResultWithPostInfo {
+            let who = ensure_signed(origin)?;
+
+            let machine_info = online_profile::Pallet::<T>::machines_info(machine_id)
+                .ok_or(Error::<T>::MachineNotFound.as_str())?;
+            ensure!(
+                machine_info.controller == who || machine_info.machine_stash == who,
+                Error::<T>::NotMachineOwner
+            );
+            EvmAddress2Account::<T>::insert(evm_address, who.clone());
```

Comment:
```suggestion
            EvmAddress2Account::<T>::insert(who.clone(), evm_address);
```
To avoid binding multiple evm addresses.
If changing the evm address is not allowed, you should check whether there is already a binding

Reply:
allow one substrate address  bond multiple evm addresses